presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkTaskExecutorFactory.java

```diff
@@ -263,11 +264,15 @@ public PrestoSparkTaskExecutorFactory(
 
         List<TaskSource> taskSources = getTaskSources(serializedTaskSources);
 
-        log.info("Task [%s] received %d splits.",
+        log.info("Task [%s] received %d splits, total size is %s.",
                 taskId,
                 taskSources.stream()
                         .mapToInt(taskSource -> taskSource.getSplits().size())
-                        .sum());
+                        .sum(),
+                succinctBytes(taskSources.stream()
```

Comment:
In theory the size could be missing. Let's log `unknown` instead of `0` in that case to avoid confusion